src/client/routes/Settings/index.tsx

```diff
@@ -12,85 +12,72 @@ import { AlertTypes, ModalName, Theme, Language } from '@types';
 import { formatPercent, getCurrentLanguage } from '@utils';
 import { ViewContainer, ThemeBox, CustomThemeButton } from '@components/app';
 import {
-  ThemesIcon,
-  WorldIcon,
-  ClockIcon,
-  SettingsIcon,
-  Icon,
+  // ThemesIcon,
+  // WorldIcon,
+  // ClockIcon,
+  // SettingsIcon,
+  // Icon,
   Button,
   ToggleButton,
   Input,
   Card,
-  CardHeader,
+  // CardHeader,
   CardContent,
   OptionList,
 } from '@components/common';
 
-const sectionsGap = '2.2rem';
-const sectionsBorderRadius = '0.8rem';
-
-const SettingsCardHeader = styled(CardHeader)`
-  margin-bottom: 1.2rem;
-`;
-
 const SettingsCardContent = styled(CardContent)`
   flex-direction: column;
   align-items: flex-start;
 `;
 
 const SettingsCard = styled(Card)`
   display: grid;
-  padding-left: 0;
-  padding-right: 0;
+  padding: ${({ theme }) => theme.card.padding} 0;
   width: 100%;
 `;
 
 const SettingsSection = styled.div`
   display: grid;
-  grid-template-columns: 18rem 1fr;
+  grid-template-columns: 15rem 1fr;
   padding: 0 ${({ theme }) => theme.card.padding};
-  grid-gap: 1.5rem;
+  grid-gap: ${({ theme }) => theme.layoutPadding};
 `;
 
 const SectionContent = styled.div`
   display: flex;
   flex-wrap: wrap;
-  grid-gap: 1.2rem;
+  grid-gap: ${({ theme }) => theme.layoutPadding};
   align-items: center;
 
   ${SettingsSection}:not(:first-child) & {
-    padding-top: ${sectionsGap};
+    padding-top: ${({ theme }) => theme.card.padding};
   }
 `;
 
-const SectionTitle = styled.div`
+const SectionTitle = styled.div<{ centerText?: boolean }>`
   display: flex;
-  align-items: flex-start;
-  fill: ${({ theme }) => theme.colors.secondary};
-  background: ${({ theme }) => theme.colors.surfaceVariantA};
-  padding: ${({ theme }) => theme.card.padding};
+  align-items: ${({ centerText }) => (centerText ? 'center' : 'flex-start')};
+  fill: currentColor;
+
+  // background: ${({ theme }) => theme.colors.background};
+  // padding: ${({ theme }) => theme.card.padding};
```

Comment:
Remove?

Reply:
Removed in 37090db0d7efc2706a2f90189ac28479f41cded6